uint8 A_position
uint8 A_current
uint8 B_position
uint8 B_current
uint8 C_position
uint8 C_current
